common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -438,7 +428,9 @@ def test_editor_visibility(self):
         """Only one editor should be visible at a time within a single response"""
         self.setup_user(roles=["Moderator"])
         self.setup_view()
-        page = DiscussionTabSingleThreadPage(self.browser, self.course_id, "comment_edit_test_thread")
+        page = DiscussionTabSingleThreadPage(
+            self.browser, self.course_id, self.discussion_id, "comment_edit_test_thread"
+        )
```

Comment:
this same call with same first 3 parameters is getting made many times, would it make sense to factor into a method where you just pass the thread_id as needed?